F21/rjuliang/LowerCaser/assets/tests.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * This is the file where we run the automatic tests
+ * See responses in a comment at the end of the file
+ */
+
+let data = [
+    {
+      "WORD": "HELLO",
+      "Ln": "en",
+      "Res": "hello"
+    },
+    {
+      "WORD": "WORLD",
+      "Ln": "en-US",
+      "Res": "world"
+    },
+    {
+      "WORD": "cAmEl",
+      "Ln": "en-IE",
+      "Res": "camel"
+    },
+    {
+      "WORD": "---OK",
+      "Ln": "en-Latn",
+      "Res": "---ok"
+    },
+    {
+      "WORD": "tAcht",
+      "Ln": "ga",
+      "Res": "t-acht"
+    },
+    {
+      "WORD": "tACHT",
+      "Ln": "ga",
+      "Res": "t-acht"
+    },
+    {
+      "WORD": "TACHT",
+      "Ln": "ga",
+      "Res": "tacht"
+    },
+    {
+      "WORD": "nAthair",
+      "Ln": "ga",
+      "Res": "n-athair"
+    },
+    {
+      "WORD": "nATHAIR",
+      "Ln": "ga",
+      "Res": "n-athair"
+    },
+    {
+      "WORD": "NATHAIR",
+      "Ln": "ga",
+      "Res": "nathair"
+    },
+    {
+      "WORD": "nÓg",
+      "Ln": "ga-IE",
+      "Res": "n-óg"
+    },
+    {
+      "WORD": "nÕg",
+      "Ln": "ga-IE",
+      "Res": "nõg"
+    },
+    {
+      "WORD": "nÕg",
+      "Ln": "ga-IE",
+      "Res": "nõg"
+    },
+    {
+      "WORD": "KASIM",
+      "Ln": "tr",
+      "Res": "kasım"
+    },
+    {
+      "WORD": "KASIM",
+      "Ln": "en",
+      "Res": "kasim"
+    },
+    {
+      "WORD": "ΠΌΛΗΣ",
+      "Ln": "el",
+      "Res": "πόλης"
+    },
+    {
+      "WORD": "官话",
+      "Ln": "zh-Hans",
+      "Res": "官话"
+    },
+    {
+      "WORD": "ภาษาไทย",
+      "Ln": "th",
+      "Res": "ภาษาไทย"
+    }
+  ]
+
```

Comment:
Giving test cases here is appreciated!

Reply:
Thank you! I like to put them usually in a comment at the end of the file